integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/AwsAccountUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.service.smartevents.integration.tests.resources;
+
+import software.tnb.aws.common.account.AWSAccount;
+import software.tnb.common.account.Accounts;
+
+public class AwsAccountUtil {
```

Comment:
Just a nitpick, isn't it enough to use name `AwsAccount`?

Reply:
yes, makes sense. Done